lib/ts/recipe/emailpassword/index.ts

```diff
@@ -199,7 +207,7 @@ export default class Wrapper {
         userContext?: Record<string, any>
     ): Promise<
         | {
-              status: "OK" | "UNKNOWN_USER_ID_ERROR" | "RESET_PASSWORD_INVALID_TOKEN_ERROR";
+              status: "OK" | "USER_BANNED_ERROR" | "UNKNOWN_USER_ID_ERROR" | "RESET_PASSWORD_INVALID_TOKEN_ERROR";
```

Comment:
no need to modify this, since token generation wont be possible in case the user is banned